src/main/java/ch/njol/skript/localization/Message.java

```diff
@@ -49,20 +49,31 @@ public class Message {
 			firstChange = false;
 		});
 	}
-	
+
 	public final String key;
 	@Nullable
 	private String value;
+	@Nullable
+	private String defaultValue;
 	boolean revalidate = true;
-	
+
 	public Message(final String key) {
 		this.key = "" + key.toLowerCase(Locale.ENGLISH);
 		messages.add(this);
 
 		if (Skript.testing() && Language.isInitialized() && !Language.keyExists(this.key))
 			Language.missingEntryError(this.key);
 	}
-	
+
+	public Message(final String key, final String defaultValue) {
```

Comment:
Oh instead of passing it as a field for the class just pass it as parameter for the getValueOrDefault method

Ex: `SKRIPT_PREFIX_MESSAGE.getValueOrDefault(SKRIPT_PREFIX)`

Reply:
hmm give me a second

Reply:
alright changed